.github/workflows/build-push-image-main.yml

```diff
@@ -0,0 +1,36 @@
+name: Tutor Build Action
+
+on:
+  push:
+    branches:
+      - 'pearson-release/olive.main'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Get current time
+        uses: josStorer/get-current-time@v2
+        id: current-time
+        with:
+          format: YYYY-MM-DD
+
+      - name: Build and push Image
+        uses: Pearson-Advance/tutor-build-image-action@0.1.1
+        with:
+          python-version: ${{ vars.BUILD_PYTHON_VERSION }}
+          tutor-version: ${{ vars.BUILD_TUTOR_VERSION }}
+          path-image-to-be-built: '/env/build/openedx'
+          repository-name: ${{ vars.BUILD_MAIN_REPOSITORY_NAME }}
+          image-name: ${{ vars.BUILD_MAIN_IMAGE_NAME }}
+          image-tag: ${{ vars.BUILD_TUTOR_VERSION }}-${{ steps.current-time.outputs.formattedTime }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-token: ${{ secrets.DOCKERHUB_TOKEN }}
+          no-docker-cache: false
```

Comment:
Does this mean that we are going to use the Docker cache? @sergivalero20

Reply:
Exactly @Squirrel18 
According to the definition that @alexjmpb made. the value to activate the cache should be false.

Reply:
We can see it [here](https://github.com/Pearson-Advance/tutor-build-image-action/blob/vue/PADV-325/action.yml#L175)
@Squirrel18